src/components/dashboard/FaceRecognition/FRStep.js

```diff
@@ -24,11 +24,11 @@ const FRStep = ({ title, isActive, status, isProcessFailed, paddingBottom, style
   //not active use grey otherwise based on status
```

Comment:
About statusIcon, spinner, iconOrSpinner variables: if shortify comparisons (e.g. someVar === / ! == true => ! /  !! someVar, someVar === undefined => !someVar etc) you can easily describe displaying logic inside main jsx block

Reply:
About paddingBottom assignment - you shouldn't modify function parameters. Declare another variable or use defaultProps (in case with component props)

Reply:
done